public/jobs.js

```diff
@@ -0,0 +1,340 @@
+async function buildJobsTable(jobsTable, jobsTableHeader, token, message) {
+  try {
+    const response = await fetch('/api/v1/jobs', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    const data = await response.json();
+    const children = [jobsTableHeader];
+    if (response.status === 200) {
+      if (data.count === 0) {
+        jobsTable.replaceChildren(...children); // clear this for safety
+        return 0;
+      } else {
+        for (let i = 0; i < data.jobs.length; i++) {
```

Comment:
much easier to just use the javascript forEach here
`data.jobs.forEach(job => {
})....`